q2_sample_classifier/tests/data/importance_chardonney.tsv

```diff
@@ -0,0 +1,327 @@
+99882	0.0005571839389159251
+4451011	0.0005367673621172981
+4458775	0.0005354610613922348
+106252	0.0005301713126190168
+128717	0.0004890847692921625
+271943	0.0004198438160258734
+237081	0.0004196113785368746
+146935	0.00040094057274993046
+28246	0.0003872820865898205
+170244	0.0003753321093973754
+394796	0.0003630769561779562
+4345285	0.0003620100294304884
+1135391	0.00035092992275626917
+198183	0.0003429082395799726
+347195	0.000321837484901028
+842941	0.00032148902937567593
+154482	0.00031620424482054
+325027	0.00031297752477390487
+4308371	0.0002897361736257693
+4442042	0.00028687561969616186
+28841	0.0002614751742022106
+4365141	0.0002592864600918602
+4463224	0.0002544671856237352
+4403653	0.00020068725358655715
+814442	0.0001854681628289824
+1135616	0.0001810963201993466
+4352233	0.0001808688941675811
+4431592	0.00017377450975371653
+72820	0.00016726377442023347
+839152	0.00016594333445879174
+816529	0.00015564865018761215
+2797395	0.0001490369165590016
+540940	0.0001427705331111527
+429048	0.00013477008207658987
+4353264	0.00013383226709210312
+4468697	0.00012362381132402674
+2874742	0.0001205792627783874
+780555	0.00011304305885473818
+133961	0.00010757469651420703
+4449609	0.00010706581367368249
+312453	0.00010371458403674598
+1843701	0.00010216080211401838
+676509	9.716320058704939e-05
+71872	9.489811410385969e-05
+4394926	9.334312563306969e-05
+812351	9.043444708379022e-05
+102915	8.785266881993596e-05
+691423	8.74479052197794e-05
+1130824	8.098909104258687e-05
+1123395	7.858913280228686e-05
+681370	7.821268709252524e-05
+188247	7.701147547233651e-05
+269125	7.427114488130107e-05
+4344371	7.171890492404162e-05
+1072784	6.810720140934897e-05
+735945	6.782583531284291e-05
+250395	6.723418532137901e-05
+4333556	6.60098361191445e-05
+191153	6.028963138919508e-05
+4456889	5.378917869303054e-05
+4333237	5.134352337296379e-05
+563359	5.108040105701027e-05
+135956	5.0241359490998746e-05
+10196	5.0241359490996184e-05
+4474485	4.604372766711962e-05
+812929	4.460367317873236e-05
+119010	4.139914348861156e-05
+197631	4.0341884019772904e-05
+568926	3.7337250253227875e-05
+4468384	3.580564835875417e-05
+4423410	3.568860455789974e-05
+3643491	3.3617092660689695e-05
+697711	3.3188666891759055e-05
+4477551	3.3188666891756934e-05
+444962	3.0255383394620722e-05
+979344	2.5212819495517268e-05
+9939	2.4891500168818583e-05
+4463108	2.4453423469918827e-05
+939545	2.128350044041883e-05
+17309	2.1077191628870952e-05
+338757	2.098349859690685e-05
+4337404	1.897042303767303e-05
+4328104	1.8243000377503335e-05
+42664	1.6809118341572044e-05
+4407979	1.344683706427589e-05
+1144015	1.3446837064275873e-05
+4380901	1.3427118134532813e-05
+562720	1.2770100264252568e-05
+4393057	1.2376844272339594e-05
+4447334	1.1787784859310052e-05
+845178	1.0706581367368972e-05
+544313	1.0371458403674174e-05
+555725	1.0085471004943226e-05
+13763	8.9648631155042e-06
+4419276	8.951412089688543e-06
+100497	6.6916133546051555e-06
+826250	6.6913819103412555e-06
+
```

Comment:
Drive by comment: Is this file supposed to have all these blank lines in it? If it does we should address this in another PR.

Reply:
No, I didn't notice those were there :laughing: they came from how I added the second column to the file.

Reply:
Something happened and my comment here seems to be gone? But adding it back for posterity. No those lines were not supposed to be there. They came about as a result of how I added the second column to the file and I didn't notice that they were still there.